recorder/src/main/java/com/focusit/jsflight/recorder/internalevent/InternalEventRecorderConfiguration.java

```diff
@@ -0,0 +1,88 @@
+package com.focusit.jsflight.recorder.internalevent;
+
+/**
+ * Created by dkolmogortsev on 03.03.17.
+ * Representing a configuration for {@link InternalEventRecorder} to be used with
+ * {@link InternalEventRecordersManager#defineFromConfiguration(InternalEventRecorderConfiguration)}
+ */
+public class InternalEventRecorderConfiguration
+{
+    private final String outputFileBaseName;
+    //Configuration defaults
+    private int maxElementsBeforeFlush = -1;
+    private int maxQueueSize = 4096;
+    private boolean storeInGzip = false;
+    private String threadName = "internal-event-storage";
+    private long rolloverInterval = 0;
```

Comment:
It seems strange to have a rolloverInterval there because you've created a special strategy for rolling files